tests/lts_compatibility.py

```diff
@@ -538,28 +547,41 @@ def add(parser):
     compatibility_report = {}
     compatibility_report["version"] = args.ccf_version
     compatibility_report["live compatibility"] = {}
-    latest_lts_version = run_live_compatibility_with_latest(args, repo, env.branch)
-    following_lts_version = run_live_compatibility_with_following(
-        args, repo, env.branch
+
+    # Compatibility with latest LTS
+    # (e.g. when releasing 2.0.1, check compatibility with existing 1.0.17)
+    latest_lts_version = run_live_compatibility_with_latest(
+        args, repo, env.branch, this_release_branch_only=False
     )
     compatibility_report["live compatibility"].update(
         {"with latest": latest_lts_version}
     )
+
+    # Compatibility with latest LTS on the same release branch
+    # (e.g. when releasing 2.0.1, check compatibility with existing 2.0.0)
+    latest_lts_version = run_live_compatibility_with_latest(
+        args, repo, env.branch, this_release_branch_only=True
+    )
+    compatibility_report["live compatibility"].update(
+        {"with latest (same LTS)": latest_lts_version}
```

Comment:
I think of the LTS as 1.x or 2.x, rather than specific builds. So, if renaming is free, I'd be tempted to call these builds `with previous LTS` and `with current LTS`, rather than `with latest`.

Reply:
Good point. I went for `with previous LTS`, `with same LTS` (which I think is less ambiguous than "current", which sounds like the most recent LTS) and `with following LTS`.